src/recordpage.h

```diff
@@ -92,13 +97,16 @@ private slots:
     void updateRecordType();
     void appendOutput(const QString& text);
     void setError(const QString& message);
+    void addKonsoleWidget();
 
     QScopedPointer<Ui::RecordPage> ui;
 
     PerfRecord* m_perfRecord;
     QString m_resultsFile;
     QElapsedTimer m_recordTimer;
     QTimer* m_updateRuntimeTimer;
+    KParts::ReadOnlyPart* m_konsolePart = nullptr;
+    QTemporaryFile* m_konsoleFile;
```

Comment:
either `= nullptr` it here, or make it a `std::unique_ptr` (which would help with the `.reset()` I proposed